Resources.xaml.cs

```diff
@@ -1,9 +1,44 @@
 namespace Lab6_Starter;
 
+/*
+ * This class is the background functionality/methods for the resources page.
+ * This class references an object created in Link.cs in order to allow the hyperlinks to function.
+ * Author: Krystal Schneider & Taylor Showalter
+ * Date: October 30, 2024
+ */
+
 public partial class Resources : ContentPage
 {
     public Resources()
     {
         InitializeComponent();
+        LoadLinks(); // Load the list of links when the page is initialized
+    }
+
+    // Method to create and set the links as the ItemsSource for the CollectionView
+    private void LoadLinks()
+    {
+        // Creating a list of Link objects to represent resources
+        var links = new List<Link>
+    {
+        new Link("Department of Transportation Home", "https://wisconsindot.gov/Pages/Home.aspx"),
```

Comment:
This really ought to be in a file and read in from the projects resources rather than hardcoded here ...